Mathlib/Topology/Instances/ENNReal.lean

```diff
@@ -76,7 +76,7 @@ theorem tendsto_coe {f : Filter α} {m : α → ℝ≥0} {a : ℝ≥0} :
     Tendsto (fun a => (m a : ℝ≥0∞)) f (𝓝 ↑a) ↔ Tendsto m f (𝓝 a) :=
   isEmbedding_coe.tendsto_nhds_iff.symm
 
-@[fun_prop]
+@[continuity, fun_prop]
```

Comment:
What is still using `continuity` that you needed to add this?

Reply:
I guess, I started this PR a long time ago when `fun_prop` was rarely used in Mathlib. Let me try to revert this line.

Reply:
```suggestion
@[fun_prop]
```